.travis.yml

```diff
@@ -1,24 +1,17 @@
 language: java
 sudo: required
-dist: trusty
+dist: xenial
```

Comment:
will not work
xenial doesn't contain oraclejdk8

Reply:
I see, let me revert it.